core/sql/udrserv/udrserv.cpp

```diff
@@ -236,7 +238,27 @@ void InitializeJavaOptionsFromEnvironment(LmJavaOptions &javaOptions
    int maxHeapEnvvarMB = DEFAULT_JVM_MAX_HEAP_SIZE;
    sprintf(maxHeapOption, "-Xmx%dm", maxHeapEnvvarMB);
    javaOptions.addOption((const char *)maxHeapOption, TRUE);
-   
+
```

Comment:
You talked in the conversation thread about avoiding duplication of code. Selva and I also talked about that. The Language manager has the nice feature of being able to pass JVM options from the outside. If we allow this for JavaObjectInterface::createJVM() as well, couldn't we then combine the code? If I ever have time to do this, I'll attempt it, but right now it doesn't look like that will happen anytime soon.

For now, this looks good to me. If you like we can file a JIRA to combine JVM startup and to call JavaObjectInterface::createJVM() in the language manager instead of using its own code. Note that we may call this LanguageManager code from the executor as well, although it's probably highly unlikely right now.